pkgs/development/libraries/linenoise/create-pkg-config-file.sh

```diff
@@ -0,0 +1,13 @@
+cat <<EOF > linenoise.pc
+prefix=$out
+exec_prefix=\${prefix}
+libdir=\${exec_prefix}/lib
+includedir=\${prefix}/include
+
+Name: linenoise
+Description: A minimal, zero-config, BSD licensed, readline replacement.
+Requires: 
+Version: 1.0.10
+Cflags: -I\${includedir}/ \${prefix}/src/linenoise.c
```

Comment:
Having a C source in `Cflags` strikes me as odd. Is that even allowed? Wouldn't this lead to linenoise.c being compiled into every compilation unit that uses the Cflags from linenoise.pc?